src/js/components/FileInput/__tests__/__snapshots__/FileInput-test.tsx.snap

```diff
@@ -1207,6 +1206,7 @@ exports[`FileInput disabled 1`] = `
       </span>
       <a
         class="c6"
+        disabled=""
```

Comment:
Was expecting this to be a true/false value rather than an empty string. Do you know why it's getting this value